Formula/node@4.rb

```diff
@@ -20,13 +21,11 @@ class NodeAT4 < Formula
   option "without-completion", "npm bash completion will not be installed"
   option "with-full-icu", "Build with full-icu (all locales) instead of small-icu (English only)"
 
-  depends_on "python@2" => :build if MacOS.version <= :snow_leopard
```

Comment:
We should leave these in the formulae for now. I don't want start mass removing these wherever we think it may not actually work

Reply:
But the build with `python@2` (for OS X 10.6-) does not work as it is right now, because the gyp used to build node won't finde our python, because 1. it's not linked anymore, 2. its called `python2` and not just `python`.

To fix the build on these old system we would basically need to create a symlink to our `python2` executable called just `python` in the `buildpath` and prepend this to the `$PATH`.
Do you want me to do this?

Reply:
No it doesn't need to be fixed. These are primarily placeholders that indicate the formula uses python.

Reply:
ok, reverted the `depends_on "python@2" => :build if MacOS.version <= :snow_leopard` removal in both PRs

Reply:
thanks!